CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingPageTracker.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  OnboardingPageTracker.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import Foundation
+
+class OnboardingPageTracker {
+    private var pages: [OnboardingPageModel]
+
+    init(pages: [OnboardingPageModel]) {
+        self.pages = pages
+    }
+
+    func isPageNotSeen(_ page: OnboardingPageModel) -> Bool {
+      pages.contains(page)
```

Comment:
[Suggestion] @ValentinaIancu-Gini please doublecheck spacing for this line

Reply:
Done. Thank you!